packages/react-components/lib/task-summary/task-summary-accordion.tsx

```diff
@@ -52,11 +52,16 @@ export const TaskSummaryAccordion = React.memo((props: TaskSummaryAccordionProps
   const [expanded, setExpanded] = React.useState<string[]>([]);
   const [selected, setSelected] = React.useState<string>('');
 
-  const handleToggle = (event: React.ChangeEvent<Record<string, unknown>>, nodeIds: string[]) => {
+  /**
+   * Here exists a type conflict, the linter told us to use Record<string, unknown> if we use `{}` as
+   * it's on the type definition, but on build time we get this error:
+   * Type '(event: React.ChangeEvent<Record<string, unknown>>, nodeIds: string) => void' is not
+   * assignable to type '((event: ChangeEvent<{}>, nodeIds: string) => void) | ((event: ChangeEvent<{}>, nodeIds: string[]) => void) | undefined'.
+   */
+  const handleToggle = (event: React.ChangeEvent<unknown>, nodeIds: string[]) => {
```

Comment:
We can disable the linter for this line, or see if you can work around it by explicitly specifying the type.

```ts
const handleToggle: SomeType = (event, nodeIds) => {}
```